pkgs/development/python-modules/libcst/default.nix

```diff
@@ -1,35 +1,46 @@
-{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder, black, isort
-, pytestCheckHook, pyyaml, typing-extensions, typing-inspect, dataclasses }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, hypothesis
+, doCheck ? true
+, dataclasses
+, hypothesmith
+, pytestCheckHook
+, pyyaml
+, typing-extensions
+, typing-inspect
+, black
+, isort
+}:
 
 buildPythonPackage rec {
   pname = "libcst";
-  version = "0.3.13";
+  version = "0.3.17";
 
   # Some files for tests missing from PyPi
   # https://github.com/Instagram/LibCST/issues/331
   src = fetchFromGitHub {
     owner = "instagram";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0pbddjrsqj641mr6zijk2phfn15dampbx268zcws4bhhhnrxlj65";
+    sha256 = "sha256-mlSeB9OjCiUVYwcPYNrQdlfcj9DV/+wqVWt91uFsQsU=";
   };
 
   disabled = pythonOlder "3.6";
 
-  propagatedBuildInputs = [ pyyaml typing-inspect ]
+  propagatedBuildInputs = [ hypothesis typing-inspect pyyaml ]
     ++ lib.optional (pythonOlder "3.7") dataclasses;
 
-  checkInputs = [ black isort pytestCheckHook ];
+  checkInputs = [ black hypothesmith isort pytestCheckHook ];
+
+  inherit doCheck;
 
   # https://github.com/Instagram/LibCST/issues/346
```

Comment:
This issue is about the circular dependency. Now that that's no longer a problem here, can this reference be removed?

Reply:
Good catch! Thank you very much.